dags/gmail_data_import_pipeline.py

```diff
@@ -249,5 +299,12 @@ def gmail_thread_details_etl(**kwargs):
 
 # pylint: disable=pointless-statement
 # define dependencies between tasks in the DAG
-get_data_config_task >> [gmail_label_data_etl_task, gmail_link_message_thread_ids_etl_task]
-gmail_link_message_thread_ids_etl_task >> gmail_thread_details_etl_task
+get_data_config_task >> [
+                            gmail_label_data_etl_task,
```

Comment:
minor: I would personally reduce the indentation and put a blank line between after `]` to make it easier to see that it is a separate statement.